### ISSUE TITLE
Custom Latex Modding

### ISSUE THREAD

**Issue by GodkingLuckyBunny** [LtxProgrammer/Changed-Minecraft-Mod#1011]
I currently have no idea how to mod in custom latex beasts, and I am really struggling to figure it out. If a grander documentation on how it's done (with examples, too, pls) or an easier way to mod in new Latex Beasts was made, that'd be great. I'd love to mod in some Latex Beasts, or creatures that are similar from other games.

**Comment by Foxyas** [comment]
https://changedmc.miraheze.org/wiki/API

**Comment by GodkingLuckyBunny** [comment]
Yes, I'm aware of the wiki, but...I don't know what it's saying to do... I might just be dumb, but it feels..confusing to my brain. Might be the WIP state of the wiki, but idk.

**Comment by spacedan228** [comment]
the wiki should be updated, but no one really had the will to do that

**Comment by GodkingLuckyBunny** [comment]
Yeah, I figured at this rate. (Also, thanks for adding the label, Ltx, lovely mod.) I hope that, when that mod is in a further state of development, it'll get the documentation updated, and we can all much better understand it. I know it's likely not being updated much rn because of everything being subject to change, but it'd be nice for it to be up to date when possible. I also hope ways for custom entities that don't require Forge MDK to make will exist at some point, though I know an MCreator plugin was attempted

**Comment by gengyoubo** [comment]
I have an idea, would it be possible to make a MCreator-like program (and then have it come with a documentation) to make it easier?

**Comment by gengyoubo** [comment]
https://github.com/gengyoubo/LTX-Customizer

**Comment by gengyoubo** [comment]
> I currently have no idea how to mod in custom latex beasts, and I am really struggling to figure it out. If a grander documentation on how it's done (with examples, too, pls) or an easier way to mod in new Latex Beasts was made, that'd be great. I'd love to mod in some Latex Beasts, or creatures that are similar from other games.

Modifying should be difficult, if it's creating it's worth a try.

**Comment by LtxProgrammer** [comment]
Wiki has been updated with a [tutorial](https://github.com/LtxProgrammer/Changed-Minecraft-Mod/wiki/Tutorials:-Creating-a-Simple-Transfur).
